signalfx/config/config.proto

```diff
@@ -159,5 +159,13 @@ message Params {
 		// The span tag that is used as the value of the remoteEndpoint.ipv4
 		// field of the span sent to SignalFx
 		string remote_endpoint_ip_tag_key = 6;
+
+        // If true, the local and remote endpoints will be swapped for
```

Comment:
Is there a reason why you wouldn't want to do this?

Reply:
The fact that I don't know makes me want to make it configurable as an "escape hatch" for some unforeseen use of tracespans or if they ever have more advanced/dynamic attribute expressions that might make it unnecessary to do the swapping.